### ISSUE TITLE
Common setup for logging for all virtual servers

### ISSUE THREAD

**Issue by danielniccoli** [cherokee/webserver#465]
_Original author: christia...@gmail.com (May 12, 2009 18:23:13)_

It'd be nice to have a way of supplying a common configuration for logging
and then being able to use that for virtual servers. Now, logging for all
my virtual servers is identical (to log file, Apache-compatible, etc.)
except for the filename, which I have to remember to change by hand every
time I create a new virtual server, even though the filename is always
/var/log/cherokee/${VIRTUAL_SERVER_NAME}-access (and -error).

It'd be nice to be able to specify all that only once and have all new
virtual servers use that configuration by default. One would need %x or
$variables or something to be able to insert the name of the virtual
server, etc.

_Original issue: http://code.google.com/p/cherokee/issues/detail?id=461_
